drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-backend/src/test/java/org/drools/workbench/screens/guided/dtable/backend/server/GuidedDecisionTableApplicationPreferencesLoaderTest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.drools.workbench.screens.guided.dtable.backend.server;
+
+import org.drools.workbench.screens.guided.dtable.service.GuidedDecisionTableEditorService;
+import org.junit.After;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class GuidedDecisionTableApplicationPreferencesLoaderTest {
+
+    @After
+    public void tearDown() throws Exception {
+        System.clearProperty( GuidedDecisionTableEditorService.DTABLE_VERIFICATION_ENABLED );
+    }
+
+    @Test
+    public void notSet() throws Exception {
+        final GuidedDecisionTableApplicationPreferencesLoader loader = new GuidedDecisionTableApplicationPreferencesLoader();
+
+        assertNull( new GuidedDecisionTableApplicationPreferencesLoader().load().get( GuidedDecisionTableEditorService.DTABLE_VERIFICATION_ENABLED ) );
```

Comment:
OK.. you have a test for ```null``` so you must handle ```null``` elsewhere.... probably find as I scroll down more...